app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -2,25 +2,14 @@ package com.uc.mymovies
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.util.Log
-import android.widget.ListAdapter
-import android.widget.ListView
-import com.google.firebase.firestore.FirebaseFirestore
-import com.google.firebase.firestore.FirebaseFirestoreSettings
-import com.uc.mymovies.service.MovieService
+import com.uc.mymovies.ui.main.MainViewModel
 
+/**
+ * MainActivity, responsible for UI/fragment functions
+ */
 class MainActivity : AppCompatActivity() {
 
-    var movieService: MovieService = MovieService()
-
-    init {
-        fetchMovies()
-    }
-
-    private fun fetchMovies() {
-       var result = movieService.fetchUserMovies("connorkeith")
-        Log.d("TEST-LOG: ", result.toString())
-    }
```

Comment:
Data should not be accessed directly in the View/UI; keep presentation logic in the viewmodel.